message_ix_models/model/material/__init__.py

```diff
@@ -657,7 +656,7 @@ def add_data_2(scenario, dry_run=False):
     log.info("done")
 
 
-@cli.command("modify_cost")
+@cli.command("modify_cost", hidden=True)
```

Comment:
In general, we prefer hyphens to underscores for CLI command and option names and values, e.g. `modify-cost`.